src/main/kotlin/com/openlattice/organizations/OrganizationMetadataEntitySetsService.kt

```diff
@@ -203,72 +203,79 @@ class OrganizationMetadataEntitySetsService(private val edmService: EdmManager)
         )
     }
 
-    fun addDatasetColumn(entitySet: EntitySet, propertyType: PropertyType) {
+    fun addDatasetColumn(entitySet: EntitySet, propertyTypesInEntitySet: Collection<PropertyType>) {
```

Comment:
would also be good to rename these both to `addDatasetColumn` while we're making updates here in all the projects anyway

Reply:
:+1:

Reply:
I assumed you meant addDatasetColumns here.